beacon_node/beacon_chain/src/block_verification_types.rs

```diff
@@ -144,6 +149,35 @@ impl<E: EthSpec> RpcBlock<E> {
         })
     }
 
+    pub fn new_with_custody_columns(
+        block_root: Option<Hash256>,
+        block: Arc<SignedBeaconBlock<E>>,
+        custody_columns: Vec<CustodyDataColumn<E>>,
+    ) -> Result<Self, AvailabilityCheckError> {
+        let block_root = block_root.unwrap_or_else(|| get_block_root(&block));
+
+        if let Ok(block_commitments) = block.message().body().blob_kzg_commitments() {
+            // The number of required custody columns is out of scope here.
+            if !block_commitments.is_empty() && custody_columns.is_empty() {
+                return Err(AvailabilityCheckError::MissingCustodyColumns);
+            }
+        }
+        // Treat empty blob lists as if they are missing.
+        let inner = if custody_columns.is_empty() {
```

Comment:
This should be `!custody_columns.is_empty()` but it looks like we can just remove this method as it doesn't seem to be used anywhere?

Reply:
Method removed in Addressed in #5748